tests/lsblk.spec.js

```diff
@@ -220,6 +208,137 @@ describe('Drivelist', function() {
 
     });
 
+    it('can do apris', function() {
```

Comment:
Is `apris` a typo?

Reply:
🤦‍♂️  that entire test description is a typo